docs/spark-connector/spark-catalog-iceberg.md

```diff
@@ -12,31 +12,35 @@ The Gravitino Spark connector offers the capability to read and write Iceberg ta
 
 #### Support DML and DDL operations:
 
-- `CREATE TABLE` 
- 
-Supports basic create table clause including table schema, properties, partition, does not support distribution and sort orders.
-
+- `CREATE TABLE`
+  - `Supports basic create table clause including table schema, properties, partition, does not support distribution and sort orders.`
 - `DROP TABLE`
 - `ALTER TABLE`
 - `INSERT INTO&OVERWRITE`
 - `SELECT`
 - `MERGE INOT`
 - `DELETE FROM`
 - `UPDATE`
+- `CALL`
+- `TIME TRAVEL`
+- `DESCRIBE SCHEMA AND RESERVED PROPERTIES`
```

Comment:
it's hard to understand `RESERVED PROPERTIES`, I prefer to remove this

Reply:
done